docs/advanced/plugin/governance-plugins.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Governance Plugins
+sidebar_label: Governance Plugins
+sidebar_position: 1
+---
+
+A DAO can be understood as an arbitrary executor, a system or entity capable of performing a wide range of actions based on permissions it holds. Governance plugins are essential components of your DAO, allowing it to execute actions and manage its operations.
+
+While a DAO can technically function without a governance plugin by granting the `EXECUTE_PERMISSION_ID` to an EOA, this practice is generally discouraged as it defeats the purpose of decentralized governance.
+
+### Execute Permission
+
+The `EXECUTE_PERMISSION_ID` is one of the most critical permissions within your DAO. It allows the entity holding this permission, whether an EOA or a smart contract, to call the DAO's `execute` function. This function is essential for carrying out decisions and actions within the DAO.
+
+### Governance Plugin
+
+Governance plugins are characterized by the **ability to execute actions in the DAO** they have been installed on. Accordingly, the `EXECUTE_PERMISSION_ID` is granted to the governance plugin contract upon installation in the DAO. If using Aragon OSx protocol (DAOFactory `createDao` function) at least one plugin installation is required when deploying the DAO.
```

Comment:
I would say something in the lines

Goverannce plugins should be able to execute actions on behalf of the dao so the EXECUTION_PERMISSION must be granted to this type of plugins via the PluginSetup. 

The line about the createDao I'm not sure it fits here